xen-ocaml/bindings/main.c

```diff
@@ -26,6 +27,11 @@ int errno;
 static char *argv[] = { "mirage", NULL };
 static unsigned long irqflags;
 
+/* XXX TODO: keep in sync with mirage-xen-minios */
+void setup_xen_features(void);
+void init_console(void);
+void init_gnttab(void);
+
```

Comment:
I think you can just include the minios headers here (`console.h` and `gnttab.h`).